gdeep/trainer/trainer.py

```diff
@@ -139,7 +141,9 @@ def __init__(
         writer: Optional[SummaryWriter] = None,
         training_metric: Optional[Callable[[Tensor, Tensor], float]] = None,
         k_fold_class: Optional[BaseCrossValidator] = None,
+        print_every: Optional[int] = 1,
```

Comment:
@AnthoJack if you use `Optional`, then it is expected be nullable. In this case you default it to 1 and then check directly if it is a valid int.

I would remove completely the `Optional` keyword.

Reply:
Very well. I'll do that right away